src/deps.txt

```diff
@@ -9,13 +9,9 @@ sudo
 dumb-init
 
 # For composes
-rpm-ostree
+rpm-ostree createrepo_c dnf-utils openssh-clients
```

Comment:
Should we remove walters-buildtools-fedora-fedora-28.repo  copr repo as well which gets added at https://github.com/coreos/coreos-assembler/blob/master/build.sh#L35 ?